### PR TITLE
ANT-2942

### PR DESCRIPTION
**Pull request by eleon-chata-ai**
- sorting-not-working-on-pivot-table

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,6 +81,7 @@
     "lodash.has": "^4.5.2",
     "lodash.isempty": "^4.4.0",
     "lodash.isequal": "^4.5.0",
+    "luxon": "^3.5.0",
```
Comment (Shiv-Chata) [inline package.json:84]: What is this pkg being used for? Is it still being used or can it be removed?

File: src/components/ChataTable/TableWrapper.js [hunk 2]
```diff
@@ -85,6 +85,7 @@ export default class TableWrapper extends React.Component {
   instantiateTabulator = () => {
     // Instantiate Tabulator when element is mounted
     this.tabulator = new Tabulator(this.tableRef, {
+      debugInvalidOptions: false,
```
Comment (Shiv-Chata) [inline src/components/ChataTable/TableWrapper.js:88]: Is this the right solution to the problem of those console logs warnings showing up? Or should could we handle the invalid options better?

File: package.json (hunk 1 above)
Comment (eleon-chata-ai) [reply package.json:84]: tabulator lib we are using for tables requires luxon and type/luxon for date sorting 
Dependency Required
You will need to include the luxon.js library to use this sorter

**Review by eleon-chata-ai — COMMENTED** [review]
> Is this the right solution to the problem of those console logs warnings showing up? Or should could we handle the invalid options better?  

From reading online, this is what most have suggested as it was an issue in 5.4.4 Tabulator. But I'll pull the pr back and see what other options there are for both fixes

**Pull request closed without merging**